lib/datahub-client/data_platform_catalogue/client/search.py

```diff
@@ -256,16 +255,12 @@ def _parse_result(
         last_modified = parse_last_modified(entity)
         name, display_name, qualified_name = parse_names(entity, properties)
 
-        relations = parse_relations(
-            RelationshipType.PARENT, entity.get("relationships", {})
-        )
         domain = parse_domain(entity)
 
         metadata = {
             "owner": owner.display_name,
             "owner_email": owner.email,
             "total_parents": entity.get("relationships", {}).get("total", 0),
-            "parents": relations[RelationshipType.PARENT],
```

Comment:
how come this is removed? Is this because we're not using it on the search results page?

Reply:
It was all geared towards data products in the search query, which i've removed. We didn't use the information anyway but i think one for another issue if we want to capture parent containers and present in the search results